package.json

```diff
@@ -14,7 +14,7 @@
   "//": "...",
   "license": "MIT",
   "engines": {
-    "node": ">=20.6.1"
+    "node": ">=18.18.0"
```

Comment:
nodejs v18? semantic-release requires v20 if I recall correctly?

Reply:
That's correct, but our test matrix has 18 version so I thought maybe you would like to support 18 as well, I have reverted the changes!